.github/workflows/build_and_push_to_docker.yml

```diff
@@ -36,13 +31,12 @@ jobs:
           export RENKU_PIP_SPEC="renku==${RENKU_VERSION}"
         fi
 
-        export DOCKER_TAG="renku$RENKU_VERSION-$LABEL"
         docker build docker/py \
           --build-arg RENKU_PIP_SPEC=${RENKU_PIP_SPEC} \
-          --tag $DOCKER_NAME-py:3.7-$DOCKER_TAG
-        docker push $DOCKER_NAME-py:3.7-$DOCKER_TAG
+          --tag $DOCKER_NAME-py:3.7-$LABEL
+        docker push $DOCKER_NAME-py:3.7-$LABEL
       env:
-        RENKU_VERSION: ${{ matrix.RENKU_VERSION }}
+        RENKU_VERSION: "0.10.4"
```

Comment:
i thought we were taking the renku version completely out of the build, and therefore we'd also want to remove the pipx installation from these files (because I thought i saw in `renku-project-template` we were including the renku installation in the Dockerfile there?)

Reply:
I was in favour of completely taking it out, but I think the majority agreement was to always have a Renku version installed in the images.

Reply:
I think renku should be installed - but we don't have to be extra diligent about keeping the versions in lock-step since the proper version will be installed during the build anyway.

Reply:
wait i thought we agreed we don't need to specify a version at all in the build